setup.cfg

```diff
@@ -0,0 +1,51 @@
+[metadata]
+name = binwalk
+author = Craig Heffner
+author-email = info@refirmlabs.com
+description = Firmware Analysis Tool
+long_description = file: README.md
+home-page = https://github.com/ReFirmLabs/binwalk
+keywords = firmware, binary, analysis, reverse-engineering, signature
+license = MIT
+license-file = LICENSE
+platform = any
+
+classifiers =
+   Programming Language :: Python
+   Programming Language :: Python :: 2
+   Programming Language :: Python :: 3
+   Development Status :: 4 - Beta
+   Environment :: Other Environment
+   Intended Audience :: Developers
+   Intended Audience :: Researchers
+   License :: OSI Approved :: MIT
+   Operating System :: OS Independent
+
+[options]
+zip_safe = true
+include_package_data = true
+python_requires = >= 2.7, != 3.0.*, != 3.1.*, != 3.2.*
+#, != 3.3.*
+setup_requires = 
+    setuptools
+install_requires = 
+    setuptools
+
+#test_suite = tests.tests.Tests
+dependency_links = ['git+https://github.com/devttys0/yaffshiv.git']
+#'git+https://github.com/devttys0/sasquatch.git',
+
+[options.extras_require]
+plots = matplotlib
+x86 = capstone
+YAFFS = yaffshiv
```

Comment:
should be ```yaffshiv @ git+https://github.com/devttys0/yaffshiv.git``` but currently it doesn't work because of a bug in pip not fixed even in `master`.